var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -87,7 +90,7 @@ def install(self, spec, prefix):
             make()
             make("install")
 
-    @when('@:8.1')
+    @when('@:8.1')  # noqa
```

Comment:
I've also seen this redefinition complaint a lot lately. We should add this to `.flake8` and ignore it permanently.